zanata-war/src/main/java/org/zanata/security/SecurityFunctions.java

```diff
@@ -66,29 +64,34 @@
  *         href="mailto:camunoz@redhat.com">camunoz@redhat.com</a>
  */
 @Slf4j
-public class SecurityFunctions {
-    protected SecurityFunctions() {
-    }
+public class SecurityFunctions extends PermissionProvider {
+
+    @Inject
+    private ZanataIdentity identity;
+
+    @Inject
+    @Authenticated
+    private HAccount authenticatedAccount;
```

Comment:
Perhaps we should add `@Nullable` here for clarity. And (while we're on the subject) maybe on the producer (`ZanataJpaIdentityStore.getAuthenticatedAccount()`), which is implicitly `@Dependent` (I think explicit would be better).